airflow/www/templates/airflow/dags.html

```diff
@@ -477,5 +477,9 @@ <h2>DAGs</h2>
           container: "body",
         });
       });
+      function urlWithOrigin(url) {
+        url = url + '&origin=' + encodeURIComponent(window.location);
+        window.location = url;
```

Comment:
What's going on here - why do we need this function and to change the link from a normal one to one requiring Javascript?

Reply:
Can we use [`request.path`](http://flask.pocoo.org/docs/1.0/api/#flask.Request.path) instead?

Reply:
Looks like `origin={{ request.base_url }}` fits here. Thanks for your suggestion. I'm going to change it.